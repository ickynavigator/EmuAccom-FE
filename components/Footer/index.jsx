import { Button, Group, Navbar, Text } from "@mantine/core";
import React from "react";
import styles from "../../styles/components/Footer.module.scss";

export const Footer = () => {
  const year = new Date().getFullYear();

  return (
    <Navbar component="footer" height={120} className={styles.footer}>
      <Group
        sx={{ height: "100%", justifyContent: "center" }}
        px={10}
        my={10}
        position="apart"
        align="center"
      >
        <Text>
          EmuAccom - Group 5 <span>&copy; {year}</span>
        </Text>

        <Group sx={{ justifyContent: "center" }}>
          <Button
            component="a"
            href="https://github.com/ickynavigator/EmuAccom-FE"
            target="_blank"
            rel="noopener noreferrer"
          >
            Frontend source code
          </Button>
          <Button
            component="a"
            href="https://github.com/ickynavigator/EmuAccom-BE"
            target="_blank"
            rel="noopener noreferrer"
          >
            Backend source code
          </Button>
        </Group>
      </Group>
    </Navbar>
  );
};
export default Footer;
