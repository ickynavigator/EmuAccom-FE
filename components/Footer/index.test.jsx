import { render } from "@testing-library/react";
import React from "react";
import Footer from "./index";

describe("Footer component tests", () => {
  it("matches the Snapshot", () => {
    const { container } = render(<Footer />);
    expect(container).toMatchSnapshot();
  });
});
