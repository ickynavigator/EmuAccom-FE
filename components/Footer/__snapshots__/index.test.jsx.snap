// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component tests matches the Snapshot 1`] = `
<div>
  <footer
    class="footer"
  >
    <a
      href="/"
      rel="noopener noreferrer"
    >
      Emu Accom
    </a>
    <a
      href="#"
      rel="noopener noreferrer"
      target="_blank"
    >
      Github source code
    </a>
  </footer>
</div>
`;
